cfme/js.py

```diff
@@ -7,7 +7,7 @@
 
 return {
     jquery: jQuery.active,
-    prototype: Ajax.activeRequestCount,
+    prototype: (typeof Ajax == "undefined") ? 0 : Ajax.activeRequestCount,
```

Comment:
Are you sore you want to be so paranoid? typeof returns string, undefined is a string so this should not make troubles ...